src-docs/faq/compat.md

```diff
@@ -8,6 +8,8 @@ Webxdc apps run in a restricted environment, but the following practices are per
 - `localStorage`, `sessionStorage`, `indexedDB`
 - `visibilitychange` events
 - `window.navigator.language`
+- `window.location.href` but you can not specify or assume anything
+  about the scheme or domain part of the url. 
```

Comment:
idea: we could also call it `deepLink` and specify that it is always a relative path + query? + fragment? that is appended to the internal webxdc url.

like it technically opens:

webxdc_webview.open(webxdc_base_url + deepLink)

Reply:
the name can't be whatever here as we are talking about normal web API, I think it is better to just mention window.location, and say you can influentiate the initial relative location and (fragment/query) via sendUpdate and point to `update.href` docs

about saying it is a relative link, ex. './subpage.html#scoreboard' it makes sense @r10s also told me the same some hours ago

Reply:
after more discussions and objections, please re-check the update.href definition. and re-open a conversation if needed.